stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -111,15 +112,15 @@ public fun ChannelInfo(
             ChannelOption(
                 title = stringResource(id = R.string.stream_compose_channel_info_delete_conversation),
                 titleColor = ChatTheme.colors.errorAccent,
```

Comment:
This should be `painterResource`. If minor breaking change to the `ChannelOption` class is possible at this stage, then better to do it now.

Reply:
You mean we should replace the icon with a painter? Yeah sure, you can do that, but make sure to add it to the changelog! :]